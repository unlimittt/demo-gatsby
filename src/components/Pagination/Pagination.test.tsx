import React from "react"
import Enzyme, { shallow } from "enzyme"
import Adapter from "enzyme-adapter-react-16"
import Pagination from "./Pagination"

Enzyme.configure({ adapter: new Adapter() })

describe("Pagination", () => {
  const props = {
    prevPagePath: "/page/1",
    nextPagePath: "/page/3",
    hasNextPage: true,
    hasPrevPage: true,
  }

  it("renders correctly", () => {
    const wrapper = shallow(<Pagination {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
