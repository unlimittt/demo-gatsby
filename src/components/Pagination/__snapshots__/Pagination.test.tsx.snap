// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination renders correctly 1`] = `
<div
  className="pagination"
>
  <div
    className="pagination__prev"
  >
    <mockConstructor
      className="pagination__prev-link"
      rel="prev"
      to="/page/1"
    >
      ← PREV
    </mockConstructor>
  </div>
  <div
    className="pagination__next"
  >
    <mockConstructor
      className="pagination__next-link"
      rel="next"
      to="/page/3"
    >
      → NEXT
    </mockConstructor>
  </div>
</div>
`;
